src/core_plugins/metrics/public/visualizations/components/horizontal_legend.js

```diff
@@ -0,0 +1,36 @@
+import React, { PropTypes } from 'react';
+import _ from 'lodash';
+import createLegendSeries from '../lib/create_legend_series';
+
+function HorizontalLegend(props) {
```

Comment:
Composition over inheritance. Also the stateless function is suppose to be more efficient.